### PR TITLE
Fixes selectOut on weaver model queries

### PR DESCRIPTION
**Pull request by gvdent**
Fixes selectout which were previously flattened

### CODE REVIEW

File: test/WeaverModelQuery.test.coffee [hunk 1]
```diff
@@ -71,6 +77,27 @@ describe 'WeaverModelQuery test', ->
           assert.equal(p.relation('hasHead').first().constructor, model.Head)  # <- this fails currently
         )
 
+      it.skip 'should not error on livesInSomebuilding', ->
```
Comment (gvdent) [inline test/WeaverModelQuery.test.coffee]: This test errors, which according to the model it should not (Persons do live in a building).

**Comment by Gielert** [comment]
Looks good imo

File: test/WeaverModelQuery.test.coffee [hunk 2]
```diff
@@ -71,6 +77,27 @@ describe 'WeaverModelQuery test', ->
...
+
+      it 'translates selectOut correctly (whitebox testing)', ->
+        q = new Weaver.ModelQuery()
+          .restrict("personB")
+          .selectOut("Person.livesIn", "Building.placedBy")
+
+        expect(q).to.have.property('_selectOut').to.have.length.be(1)
+        expect(q._selectOut[0]).to.have.length.be(2)
+
+
```
Comment (gvdent) [inline test/WeaverModelQuery.test.coffee]: EXCESSIVE NEWLINES!

**Comment by bastbijl** [comment]
This has Gijs quality

**Pull request merged**